### ISSUE TITLE
The operating system does not start

### ISSUE THREAD

**Issue by AlekseyZaiets** [intel-aero/meta-intel-aero#417]
An update was released when working with the operating system. The update was unsuccessful. The operating system did not start and I wanted to reinstall it. The flash drive was not displayed in the BIOS and I changed some BIOS parameters. After that, the quadrocopter turns on but there is no image and no power is supplied to the keyboard. I tried to figure out how to reset everything to the initial level. Did not work out. Found a button on the board, it allows you to turn off the board. Nothing helps. help me!!!

**Comment by afarrukh-emhr23** [comment]
Which operating system are you working with? Have you tried following all the steps at this link: https://github.com/intel-aero/meta-intel-aero/wiki/02-Initial-Setup

**Comment by AlekseyZaiets** [comment]
After installing the update, the drone turned off. At startup, there is no power on the USB connector and no image output. At the same time, he takes off and reacts to the control equipment. All methods described in the instructions have already been used. It seems to me there is a problem on the BIOS chip. I want to contact the workshop to flash the BIOS or replace the chip. The processor itself does not start. I tried all the options for powering the board.

**Comment by afarrukh-emhr23** [comment]
Have you updated the BIOS using the Initial setup page on the Aero wiki for yocto linux?

**Comment by AlekseyZaiets** [comment]
No. I did not make it in time. it is possible to update the BIOS without a working keyboard and image?
